events/conftest.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from django.utils import timezone
+from django.utils.datetime_safe import datetime
+from events.class_models.event_models import Colors
+from events.class_tests.create_events import TITLE
+from reminders.class_models.reminder_models import ReminderType
+
+
+# define constants
+def pytest_configure():
+    pytest.invalid_method = "undefined"
+    pytest.current_date = datetime.now()
+    pytest.valid_method = ReminderType.EMAIL
+    pytest.message = "Joined Meeting in 50 minutes"
+    pytest.valid_date_time = datetime(
+        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 12, 0, 0, tzinfo=timezone.utc
+    )
+    pytest.invalid_date_time = datetime(
+        pytest.current_date.year - 1, pytest.current_date.month, pytest.current_date.day, 12, 0, 0, tzinfo=timezone.utc
+    )
+
+    pytest.date_time_end = datetime(
+        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 14, 0, 0, tzinfo=timezone.utc
+    )
+    pytest.date_time_start = datetime(
+        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 13, 0, 0, tzinfo=timezone.utc
+    )
```

Comment:
```suggestion
    pytest.event_date_time_end = datetime(
        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 14, 0, 0, tzinfo=timezone.utc
    )
    pytest.event_date_time_start = datetime(
        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 13, 0, 0, tzinfo=timezone.utc
    )
```

Reply:
Yeah thats another way to solve that, tnx